crowbar_framework/app/controllers/restricted_controller.rb

```diff
@@ -133,4 +133,15 @@ def load_node_or_render_not_found
   def load_node
     @node = Node.find_node_by_name_or_alias(params[:id])
   end
+
+  def upgrade
```

Comment:
You're mentioning `update` in the commit message

Reply:
Indeed, and twice no less...fixed now. Thanks!